util/fusermount.c

```diff
@@ -1599,6 +1600,10 @@ int main(int argc, char *argv[])
 	   ie For the control socket to get closed.
 	   btw We don't want to use daemon() function here because
 	   it forks and messes with the file descriptors. */
+
+        max_fd = sysconf(_SC_OPEN_MAX);
+        for(int fd=3; fd<=max_fd; fd++) { if (fd!=cfd) close(fd); }
```

Comment:
Style is a bit unusual  - the project was initiated with linux kernel style - we should follow it (there are some exceptions in example/ and test/ - needs to be fixed)? I had added a .clang-format to get that style.

And I *think* it would be better to do that in mount.c setup_auto_unmount() and fuse_mount_fusermount()?

Reply:
I mean with posix_spawn_file_actions_addclose()